.cursorrules

```diff
@@ -84,6 +84,20 @@ Documentation:
 - Use JSDoc and TypeDoc for public APIs.
 - Include examples and document error cases.
 
+Changelog management:
+
+- Follow Keep a Changelog format for all CHANGELOG.md files.
+- For release PRs, categorize "### Uncategorized" entries into:
+  - "### Added" for new features and functionality (entries starting with "feat:")
+  - "### Changed" for changes to existing functionality, dependency updates, removals (entries starting with "chore:" or breaking changes)
```

Comment:
We should use `### Removed` for removals, and we should only include dependency updates if they are breaking.